engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -30,14 +30,17 @@
  * TODO: Better naming
  */
 public class DisplayResolutionDependentFBOs extends AbstractFBOsManager {
-    public static final ResourceUrn READONLY_GBUFFER = new ResourceUrn("engine:sceneOpaque");
-    public static final ResourceUrn WRITEONLY_GBUFFER = new ResourceUrn("engine:sceneOpaquePingPong");
+    // TODO: Shift this kind of swapping functionality to a more generic SwappableFBO class
+    public static final ResourceUrn BUFFER1 = new ResourceUrn("engine:sceneOpaque1");
+    public static final ResourceUrn BUFFER2 = new ResourceUrn("engine:sceneOpaque2");
```

Comment:
Regarding the first part : Yeah, no more "scene" in any of the FBOs anymore, everything reflects gbuffer or final buffer.

I do have some use cases in mind, but they all rely on being able to add custom nodes to the renderer.